### PR TITLE
Minor updates to the development instructions.

### PR DESCRIPTION
**Pull request by csadorf**
1. Make conda recommended method for environment management.
2. Make commands copy & pastable by removing the prompt.
3. Enumerate instructions.

### CODE REVIEW

**Review by vyasr — APPROVED** [review]
Approving pending the one fix.

File: doc/support.rst [hunk 1]
```diff
@@ -23,44 +23,50 @@ All contributors must agree to the `Contributor Agreement <https://github.com/gl
...
 
-Then clone your fork and install the package from source with:
+or alternatively with `venv <https://docs.python.org/3/library/venv.html>`_:
 
 .. code-block:: bash
 
-    (signac-dev) ~ $ cd path/to/my/fork/of/signac
-    (signac-dev) signac $ pip install -e . -r requirements-dev.txt
+    python -m venv ~/envs/signac-dev
+    source ~/envs/signac-flow-dev/bin/activate
```
[the remark this answers is not in the document]
Comment (csadorf) [reply doc/support.rst]: Thx for the catch!

File: doc/support.rst [hunk 2]
```diff
@@ -23,44 +23,50 @@ All contributors must agree to the `Contributor Agreement <https://github.com/gl
...
+    cd signac/
 
-or alternatively with `conda <https://conda.io/docs/>`_:
+2. We highly recommend to setup a dedicated development environment with `conda <https://conda.io/docs/>`_:
 
 .. code-block:: bash
 
-    ~ $ conda create -n signac-dev python=3 six flake8
-    ~ $ activate signac-dev
+    conda create -n signac-dev python --file requirements-dev.txt
```
Comment (b-butler) [inline doc/support.rst]: This will not work and just just be `conda create -n signac-dev python`

File: doc/support.rst [hunk 3]
```diff
@@ -23,44 +23,50 @@ All contributors must agree to the `Contributor Agreement <https://github.com/gl
...
+    conda activate signac-dev
```
Comment (b-butler) [inline doc/support.rst]: We should give follow up comments regarding how to properly set up the environment.

**Review by b-butler — CHANGES_REQUESTED** [review]
Everything besides the conda part is good.

**Comment by csadorf** [comment]
Let's just leave it *as-is* for now.

**Pull request closed without merging**